gulpfile.js

```diff
@@ -92,17 +94,14 @@ var injectLiveDemos = function(content) {
 
 gulp.task('site:clean', function() {
   return gulp.src([
-    'site/react/core/**/*.md',
-    'site/react/grid/**/*.md',
-    'site/react/chart/**/*.md',
-    'site/react/scheduler/**/*.md',
-    'site/vue/grid/**/*.md',
-  ], { read: false })
-    .pipe(clean());
+    'site/react/**/**/*.md',
```

Comment:
`/**/**/` is the same as `/**/`, isn't it?

Reply:
Yeah, you are right!